packages/core/src/Provider/Provider.ts

```diff
@@ -1,5 +1,17 @@
+import { Percent } from '../Cardano';
+import { Tip } from '@cardano-ogmios/schema';
+
 export type HealthCheckResponse = {
   ok: boolean;
+  localNode?: {
+    ledgerTip?: Tip;
+    networkSync?: Percent;
+  };
+};
+
+export type ServicesHealthCheckResponse = {
+  ok: boolean;
+  services: HealthCheckResponse[];
```

Comment:
This doesn't match the screenshots, shouldn't it be `{name, ok}`?

Reply:
Not sure I understood the exact type mismatch? 
From the screenshots, root health check response type: `{ok, services}`, individual service health check type: `{name, ok}`

Reply:
[HealthCheckResponse](https://github.com/input-output-hk/cardano-js-sdk/pull/390/files#diff-5913afa26cddc3441a4bc0483565c14043739d6ec4c32fa357ea50345892ccffR4) seems to be `{ok, localNode}` instead of `{ok, name}`

Reply:
Well, it depends on whether the HTTP server has started with dependent services on `cardano node` (`tx-submit` or `network-info`).
But you are right, the `services: HealthCheckResponse[]` type from the root should be different than the extended `HealthCheckResponse` with `localNode`.
So the question is do we want to show optional `localNode` data from the root health check endpoint or to be visible only on individual service health check endpoint?

Reply:
Probably on individual services only. Tip on the node is not necessarily consistent with tip of the service data. For example, service dependent on ogmios is likely to be slightly ahead of the service dependent on db-sync. The benefit of having those on service level is that users can then reason about data consistency. Thinking about this, I think only 'tip' is necessary for service-level healthcheck and `networkSync?: Percent;` is probably only relevant on CardanoNode. Thoughts?

Reply:
Got it, about your thought, based on the requirements both `tip` and `networkSync` are grouped under 
`localNode?: {
          ledgerTip: lastKnownTip,
          networkSync: networkSynchronization
        }` so they will be populated only from `ogmios` dependent service `/health` as we have built-in [method](https://github.com/input-output-hk/cardano-js-sdk/pull/390/files#diff-f39a864229b1800d755f14614cd8496fe47244069e7b95d5ae851156616cf34bR114-R116) for it.

So you're thinking that `tip` should be mandatory on service level `/health` regardless the dependency type? So if it depends on db-sync only to query and map tip data from db rather than just `SELECT 1` as health check query ?

Reply:
> So you're thinking that tip should be mandatory on service level /health regardless the dependency type?

Yeah, it's out of scope of this ticket though. Let's leave them grouped and optional for now and think about it when working on ADP-2128

Reply:
41fa605dd9cf00bedd1371e10ba716464835ed66

Reply:
`localNode` can only be the response from querying the node directly. We should probably make `CardanoNode` a mandatory dependency for all Providers for this consistency check alone.